frontend/package.json

```diff
@@ -36,6 +36,7 @@
     "react-firebase-hooks": "^5.1.1",
     "react-hook-form": "^7.50.1",
     "react-html-parser": "^2.0.2",
+    "react-json-csv-convert": "^1.1.1",
```

Comment:
hey guys - i'm not sure about this one. It has only 28 weekly downloads. Compared to a package like `papaparse` which has about 2M weekly downloads. Sounds like we might have to opt for `papaparse

Please check it out here - https://www.npmjs.com/package/papaparse